### PR TITLE
Tbaut remove tab navigation

### PR DESCRIPTION
**Pull request by Tbaut**
This PR mainly removes the tabs (Account and Scanner) and sets the AccountList screen as the main screen.

It also fixes a couple things related to the navigation such as #142 :
- closes https://github.com/paritytech/parity-signer/issues/224
- closes https://github.com/paritytech/parity-signer/issues/162
- closes https://github.com/paritytech/parity-signer/issues/142

Tested on Android with Tx and message signing.

There are now 2 buttons (that's at least 1 too many) on the account list, I'll tackle this (https://github.com/paritytech/parity-signer/issues/225) in a separate PR

Question: how should I proceed to avoid committing `package-lock.json`, I read it shouldn't be in `gitignore`.. what about this annoying wizard in `package.json` ? :)

### CODE REVIEW

File: src/screens/TxDetails.js [hunk 1]
```diff
@@ -96,9 +90,6 @@ export class TxDetailsView extends React.PureComponent {
           title={this.props.sender.name}
           address={this.props.sender.address}
           chainId={this.props.sender.chainId}
-          onPress={() => {
-            this.props.onPressAccount(this.props.sender);
```
Comment (maciejhirsz) [inline src/screens/TxDetails.js]: Has there been any issues with navigation here?

(I don't think I ever wanted to navigate from tx screen to account by tapping the icon, or expected that to work that way, so I think it's fine, just curious about the reasons)

Comment (Tbaut) [reply src/screens/TxDetails.js]: There has been no issue opened for this indeed. I stumbled upon this and would not expect this either, hence the removal. I have the bad habit to tackle several issues at once :). I did my best not to shoot in too many directions for this one.

Comment (fckt) [reply src/screens/TxDetails.js]: > I have the bad habit to tackle several issues at once :)

how sweet is it

**Comment by fckt** [comment]
have you tested the build for iOS? @Tbaut

**Comment by Tbaut** [comment]
Not yet, I'll do it after https://github.com/paritytech/parity-signer/issues/225

**Comment by fckt** [comment]
not the best practice

**Pull request merged**